sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -260,6 +260,13 @@ object RapidsConf {
     .checkValue(v => v >= 0 && v <= 1, "The fraction value must be in [0, 1].")
     .createWithDefault(0.9)
 
+  val RMM_ALLOC_MAX_FRACTION = conf("spark.rapids.memory.gpu.maxAllocFraction")
```

Comment:
indentation is a little off here, or maybe it's off everywhere else ;) I believe this does follow the 4 space indentation requirement.